bootstrapping/reviews.tarql

```diff
@@ -0,0 +1,40 @@
+base  <http://weverify.eu/resource/>
+prefix owl: <http://www.w3.org/2002/07/owl#>
+prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+prefix xsd: <http://www.w3.org/2001/XMLSchema#>
+prefix puml:     <http://plantuml.com/ontology#>
+prefix schema: <http://schema.org/>
+prefix skos: <http://www.w3.org/2004/02/skos/core#>
+prefix wev: <http://weverify.eu/ontology/>
+prefix geoname: <http://sws.geonames.org/>
+
+
+construct {
+    ?claimUri schema:appearance ?appearance_url .
+    ?appearance_url 
+        schema:author           ?name ;
+        schema:caption          ?content ;
+        schema:dateCreated      ?time ;
+        schema:url              ?web_scraper_start_urlURI .
+        
+
+    ?review_url 
+        schema:itemReviewd      ?claimUri;
+        schema:reviewBody       ?content ;
+        schema:hasPart          ?evidence_url .
+
+    ?evidence_url
+        schema:about            ?web_scraper_start_urlURI ;
+        schema:dateCreated      ?timeTyped ;
+        schema:additionalType   ?additionaType .
+} where {
+    bind (uri(concat("Claim/", "misinfomation/", ?id)) as ?claimUri)
+    bind (if (?type = "fake news", uri(concat("Appearance/", STRUUID())), ?null) as ?appearance_url)
+    bind (if (?type = "debunk", uri(concat("ClaimReview/", ?id)), ?null) as ?review_url)
```

Comment:
The ClaimReview uri should include the author name in the uri. It will not break anything as is here for now but there are too many csvs with consecutive numbers to form the URI without something extra between ClaimReview/ and id #

Reply:
we don't have the author name for the claim in the data. Will include misinformation as prefix in order to avoid collisions

Reply:
we don't always have the author names but will try to include it

Reply:
what function do you use to make the names suitable for URIs

Reply:
Usually use the source name (e.g. "politifact" or "AFP%20Fact%20Check") processed with the encode_for_uri function. Exact sparql code is

bind (iri(concat("Claim/", encode_for_uri(?claimReview_author_name), "/", ?urlName)) as ?claimUri)
bind (iri(concat("ClaimReview/", encode_for_uri(?claimReview_author_name), "/", ?a, "/", ?urlName)) as ?reviewUri)

This also means that the same Claim extracted from multiple sources (which data source will definitely have) will get properly merged into the same entity.